data/items.json

```diff
@@ -2,60 +2,63 @@
   "pen": {
     "id": 0,
     "isItem": true,
+    "combinableWith": [10,12,3],
```

Comment:
Please remove all of these ‘combinableWith’ properties from items since we don’t use that anymore (we use recipes.json instead)